gdal/data/nitf_spec.xsd

```diff
@@ -131,6 +131,7 @@
                     <xs:enumeration value="integer"/>
                     <xs:enumeration value="real"/>
                     <xs:enumeration value="IEEE754_Float32_BigEndian"/>
+                    <xs:enumeration value="UnsignedInt_BigEndian"/>
```

Comment:
Since this type is parsed identically to "bitmask", we could potentially remove the "bitmask" type.  Benefits of keeping "bitmask" are that it's descriptive of how it's used in nitf_spec.xml in conjunction with bit testing (cond="MASK:2").  But drawback is having duplicate types.